test/test_ssa_direct_qmc.cc

```diff
@@ -100,5 +121,22 @@ TEST(SsaDistribution, MomentTest) {
             
     EXPECT_TRUE(KH_mu1) << os1.str();
     EXPECT_TRUE(KH_mu2) << os2.str();
+*/
+    constexpr double V_f_mu1 = n_proc;
+    constexpr double V_f_mu2 = n_proc*n_proc;
+    constexpr size_t n_events = 1000*1000;
+    try {
+        KHineq::kh_test(ssa, [](size_t j){return j;}, V_f_mu1, exact_mu1, n_events);
+    }
+    catch (std::runtime_error err) {
+        FAIL() << err.what();
+    }
+    try {
+        KHineq::kh_test(ssa, [](size_t j){return j*j;}, V_f_mu2, exact_mu2, n_events);
+    }
```

Comment:
If I remove lines 130–134 then one test could pass and the other one fail but we would just know that at least one of the two failed, right? Also, if both throw an exception we only capture the first one in that way or I am wrong?